cloudferrylib/os/actions/check_config_quota_neutron.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2015 Mirantis Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and#
+# limitations under the License.
+from cloudferrylib.base.action import action
+from cloudferrylib.utils import utils as utl
+
+
+LOG = utl.get_log(__name__)
+
+
+class CheckConfigQuotaNeutron(action.Action):
+    def run(self, info=None, **kwargs):
+        src_cloud = self.src_cloud
+        dst_cloud = self.dst_cloud
+        network_src = src_cloud.resources[utl.NETWORK_RESOURCE]
+        identity_dst = src_cloud.resources[utl.IDENTITY_RESOURCE]
+        network_dst = dst_cloud.resources[utl.NETWORK_RESOURCE]
+        tenants_src = self.get_src_tenants()
+        list_quotas = network_src.list_quotas()
+        not_custom_quotas = self.get_tenants_without_quotas(tenants_src,
+                                                            list_quotas)
+        if not not_custom_quotas:
+            LOG.warning("On SRC cloud all tenants "
+                        "have custom quotas for network")
+            LOG.warning("Difference between clouds configs "
+                        "default quotas will not calculated")
+            return
+        LOG.warning("Check configs on clouds")
```

Comment:
* Change to `LOG.info`, since contents are definitely not a warning
* Update message to something meaningful, maybe something like "Checking default quota configuration on source and destination cloud"